src/database/postgres.go

```diff
@@ -0,0 +1,131 @@
+package database
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	pb "accretional.com/semantifly/proto/accretional.com/semantifly/proto"
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgconn"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+type PgxIface interface {
+	Begin(context.Context) (pgx.Tx, error)
+	Close(context.Context) error
+	Query(ctx context.Context, sql string, args ...interface{}) (pgx.Rows, error)
+	Exec(ctx context.Context, sql string, arguments ...interface{}) (pgconn.CommandTag, error)
+}
+
+func initializeDatabaseSchema(ctx context.Context, conn PgxIface) error {
+
+	_, err := conn.Exec(ctx, `
+		CREATE TABLE IF NOT EXISTS index_list (
+			name TEXT PRIMARY KEY,
+			entry JSONB
+		)
+	`)
+	if err != nil {
```

Comment:
you got rid of/kept the wrong error message

Reply:
Fixed it